server/ColorDeconvolution/ColorDeconvolution.py

```diff
@@ -16,7 +19,17 @@ def main(args):
 
     print(args.inputImageFile)
 
-    im_input = skimage.io.imread(args.inputImageFile)
+    if args.largeImage:
+        if args.region is None:
+            raise ValueError('Region must be provided when using --largeImage')
+        ts = large_image.getTileSource(args.inputImageFile)
+        if len(args.region) != 4:
+            raise ValueError('Exactly four values required for --region')
+        im_input = ts.getRegion(format=large_image.tilesource.TILE_FORMAT_NUMPY,
+                                region=dict(zip(['left', 'top', 'width', 'height'],
+                                                args.region)))[0]
```

Comment:
You can just use this for both normal and whole-slide images. No need for this if.

For the moment, you can use this for testing if the image is a whole-slide image.